client/lib/plans/features-list.js

```diff
@@ -856,65 +856,65 @@ export const FEATURES_LIST = {
 
 	[ constants.FEATURE_SECURITY_SCANNING_JETPACK ]: {
 		getSlug: () => constants.FEATURE_SECURITY_SCANNING_JETPACK,
-		getTitle: () => i18n.translate( 'Advanced Security' ),
+		getTitle: () => i18n.translate( 'Advanced security' ),
 		getDescription: () =>
 			i18n.translate( 'Also includes daily scans for malware and security threats.' ),
 		hideInfoPopover: true,
 	},
 
 	[ constants.FEATURE_REVENUE_GENERATION_JETPACK ]: {
 		getSlug: () => constants.FEATURE_REVENUE_GENERATION_JETPACK,
-		getTitle: () => i18n.translate( 'Revenue Generation' ),
+		getTitle: () => i18n.translate( 'Revenue generation' ),
 		getDescription: () => i18n.translate( 'High-quality ads to generate income from your site.' ),
 		hideInfoPopover: true,
 	},
 
 	[ constants.FEATURE_VIDEO_HOSTING_JETPACK ]: {
 		getSlug: () => constants.FEATURE_VIDEO_HOSTING_JETPACK,
-		getTitle: () => i18n.translate( 'Video Hosting' ),
+		getTitle: () => i18n.translate( 'Video hosting' ),
 		getDescription: () => i18n.translate( '13 GB of high-speed, HD, and ad-free video hosting.' ),
 		hideInfoPopover: true,
 	},
 
 	[ constants.FEATURE_SECURITY_ESSENTIALS_JETPACK ]: {
 		getSlug: () => constants.FEATURE_SECURITY_ESSENTIALS_JETPACK,
-		getTitle: () => i18n.translate( 'Essential Security' ),
+		getTitle: () => i18n.translate( 'Essential security' ),
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.